resources/elasticache-memcacheclusters.go

```diff
@@ -85,22 +102,36 @@ func (i *ElasticacheCacheCluster) Properties() types.Properties {
 	properties.Set("ClusterID", i.clusterID)
 	properties.Set("Status", i.status)
 
-	for _, tag := range i.Tags {
-		properties.SetTag(tag.Key, tag.Value)
+	if i.Tags != nil {
+		for _, tag := range i.Tags {
+			properties.SetTag(tag.Key, tag.Value)
+		}
 	}
 
 	return properties
```

Comment:
```suggestion
	return types.NewPropertiesFromStruct(i)
```

TODO: remove everything else from the Properties() method.

Reason: libnuke now has this automatic properties generator based on the resource struct.